components/raftstore/src/store/peer.rs

```diff
@@ -793,6 +793,16 @@ where
     pub pending_request_snapshot_count: Arc<AtomicUsize>,
     /// The index of last scheduled committed raft log.
     pub last_applying_idx: u64,
+    pub max_apply_unpersisted_log_limit: u64,
+    // the minimum raft index after which apply unpersisted raft log can be enabled.
+    // we force disable apply unpersisted raft log in following 2 situation:
```

Comment:
The comment format is not consistent with others. `///` should be used and the first letter needs to be capitalized.